docs/user/ppl/admin/connectors/s3glue_connector.rst

```diff
@@ -39,9 +39,8 @@ Glue Connector Properties.
     * This parameters provides the Opensearch domain host information for glue connector. This opensearch instance is used for writing index data back and also
     * ``glue.indexstore.opensearch.uri`` [Required]
     * ``glue.indexstore.opensearch.auth`` [Required]
-        * Default value for auth is ``false``.
+        * Accepted values include ["noauth", "basicauth"]
     * ``glue.indexstore.opensearch.region`` [Required]
```

Comment:
Same question here - why is this required for basic/no auth?

Reply:
will adress at all places.